com.microsoft.mrtk.accessibility/Utilities/DescribableObject.cs

```diff
@@ -106,5 +116,34 @@ public DescribableFlag Flags
         /// "login button", "rocking chair", etc.
         /// </summary>
         public string Semantic => semantic;
+
+        #region Monobehaviour methods
+
+        private AccessibilitySubsystem accessibilitySubsystem = null;
+
+        private void Awake()
+        {
+            accessibilitySubsystem = XRSubsystemHelpers.GetFirstRunningSubsystem<AccessibilitySubsystem>();
```

Comment:
The subsystem may not be running by this point.

Reply:
recommendation on how to ensure it is running when we get around to asking?

Reply:
I think we should be okay here, unless a dev has explicitly done something like disabling the subsystem or setting a custom component's execution order to be earlier than the lifecycle manager.

In those cases, we could probably log for debugging purposes.

Or do you think there would be a scenario where describable objects are present in the scene but a dev wouldn't want the subsystem running at app launch?

Reply:
One concern about logging here is that EVERY describable object will log. That can lead to a LOT of logging depending on the number of objects in the scene.

Reply:
implemented a static flag to suppress logging after the first instance